src/main/kotlin/fi/metatavu/timebank/api/scheduler/ScheduledSynchronization.kt

```diff
@@ -26,6 +26,8 @@ class ScheduledSynchronization {
     @Scheduled(cron = "0 15 3 * * ?")
     fun scheduledSynchronization(){
         runBlocking {
+            val synchronizeDeletedTimeEntries = synchronizeController.synchronizeDeletions()
```

Comment:
Since these both synchronization methods (synchronizeDeletedTimeEntries and synchronizedEntries) require fetching the same data from Forecast, it would be a good idea to fetch the data in there and then pass that same data to both of those methods instead.

Reply:
That sound like a good idea to me.